templates/common/openstack/files/usr-local-bin-openstack-afterburn-hostname.yaml

```diff
@@ -7,4 +7,5 @@ contents:
 
     # Fetch hostname from OpenStack. Set both transient and static hostnames to
     # ensure node-valid-hostname sees the new hostname immediately.
-    /usr/bin/afterburn --provider openstack --hostname=/dev/stdout | xargs hostnamectl set-hostname
+    /usr/bin/afterburn --provider openstack --hostname=/etc/hostname
```

Comment:
I don't think this will work, btw. I don't think afterburn's new SELinux confinement allows writing to /etc either.